source/hic_hal/toshiba/tmpm366/driver/inc/tmpm366_fc.h

```diff
@@ -0,0 +1,127 @@
+/**
+ *******************************************************************************
+ * @file    tmpm366_fc.h
+ * @brief   This file provides all the functions prototypes for FC driver.
+ * @version V2.0.2.4
+ * @date    2014/09/28
+ * 
+ * (C)Copyright TOSHIBA ELECTRONIC DEVICES & STORAGE CORPORATION 2017 All rights reserved
+ *******************************************************************************
```

Comment:
This and other files should have a permissive open source license.

Reply:
Corrected the copyright messages.